src/components/Context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const { Provider, Consumer } = React.createContext(null)
+
+export { Provider, Consumer }
```

Comment:
Can we export the Context as default? That will let us export it from the package by itself, which I'd rather do in case React adds new APIs to the Context (potentially some sort of `.read()`).